src/y0/algorithm/simplify_latent.py

```diff
@@ -112,33 +147,14 @@ def transform_latents_with_parents(
         graph.remove_node(latent_node)
         graph.add_edges_from(itt.product(parents, children))
 
-        new_node = Variable(f"{latent_node}{suffix}")
```

Comment:
Is this change necessary to make this Pr work? Otherwise it’s a distracting stylistic change that should be in a different pr